docs/source/onboarding_resources/contributing/index.rst

```diff
@@ -32,16 +32,12 @@ If you want to add or modify documentation, you'll first need to clone and
 install this repository.  As always, you should do this in a clean environment.
 
 Start by opening the terminal of your choosing. If you want more information on 
-this, check out this :ref:`page on terminals <terminal_access>`. Then enter 
-the commands below. 
+this, check out this :ref:`page on terminals <terminal_access>`.
```

Comment:
It seems like you don't want to delete this last sentence, since the included section contains no text